plan/property_cols_prune.go

```diff
@@ -40,6 +40,42 @@ func (p *LogicalSelection) preparePossibleProperties() (result [][]*expression.C
 	return p.children[0].preparePossibleProperties()
 }
 
+func (p *LogicalSort) preparePossibleProperties() (result [][]*expression.Column) {
+	p.children[0].preparePossibleProperties()
```

Comment:
Why we need this since there is no variable holding the return value?

Reply:
@XuHuaiyu We have to recursively call every node's `preparePossibleProperties()` in the plan tree.